src/key-value-store/pers_low_level_db_access.c

```diff
@@ -258,7 +258,7 @@ sint_t pers_lldb_open(str_t const* dbPathname, pers_lldb_purpose_e ePurpose, boo
                DLT_LOG(persComLldbDLTCtx, DLT_LOG_INFO,
                        DLT_STRING(LT_HDR); DLT_STRING(__FUNCTION__); DLT_STRING(": semaphore already exists: "); DLT_STRING(strerror(error)));
                //try to open existing semaphore
-               pLldbHandler->kissDb.kdbSem = sem_open(pLldbHandler->kissDb.semName, O_CREAT, 0644, 0);
+               pLldbHandler->kissDb.kdbSem = sem_open(pLldbHandler->kissDb.semName, O_CREAT, 0644, 1);
```

Comment:
Despite of comment above (try to open existing semaphore), sometimes the semaphore does not exist when set_open is called, in this case it will be created with incorrect 0 value.

Reply:
If the tests are still running successfully, I think we can accecpt the suggested modification.
Did you run the tests?